resources/views/livewire/oauth-clients.blade.php

```diff
@@ -0,0 +1,333 @@
+<div>
+    <div class="panel panel-default">
+        <div class="panel-heading">
+            <div style="display: flex; justify-content: space-between; align-items: center;">
+                <h2>
+                    OAuth Clients
+                </h2>
+                @if($authorizationError)
+                    <div class="alert alert-danger">
+                        <p><strong>Whoops!</strong> Something went wrong!</p>
+                        <br>
+                        {{ $authorizationError }}
+                    </div>
+                @endif
+
+                <a class="button button-small"
+                   wire:click="$emit('openModal')"
+                   onclick="$('#modal-create-client').modal('show');"
+                >
+                    Create New Client
+                </a>
+            </div>
+        </div>
+
+        <div class="panel-body">
+            <!-- Current Clients -->
+            @if($clients->count() === 0)
+                <p class="m-b-none">
+                    You have not created any OAuth clients.
+                </p>
+            @endif
+
+            @if($clients->count() > 0)
+                <table class="table table-borderless m-b-none">
```

Comment:
`Edit` and `Delete` were text before but `Delete` got changed to a button in this PR. I think we should either go with both text or both buttons (which would match what we do in tables elsewhere).

I also noticed the normal cursor "I'm a clickable thing" went away.

Before:
![before](https://github.com/snipe/snipe-it/assets/1141514/39dc9d77-035d-4caa-be76-0b3c6bb6f87a)

After:
![after](https://github.com/snipe/snipe-it/assets/1141514/a704b330-e7a2-4d1f-baa9-c4dac29d5a9f)

Reply:
AH! damn, you're right, I fixed it in the other one and completely forgot about it here.